spec/clients_spec.rb

```diff
@@ -130,4 +130,149 @@
       expect(resp[:usage]).to eq(10)
     end
   end
+
+  context 'when calling request helpers' do
```

Comment:
Inside this testing context, we are testing the APIs of AppFigures so I would prefer use more specific term instead of **request helpers**.